services/skus/service.go

```diff
@@ -2232,6 +2254,126 @@ func (s *Service) processSubmitReceipt(ctx context.Context, req model.ReceiptReq
 	return rcpt, nil
 }
 
+const errUnknownAction = model.Error("skus: unknown radom action")
+
+func (s *Service) processRadomEvent(ctx context.Context, event *radom.Event) error {
```

Comment:
question/suggestion: Other similar methods operate in terms of notifications, even though, for instance, Stripe is clear in calling it an event. I did so purposefully – for the sake of consistency. Would you consider making this consistent with other places? Those methods were written not so long ago, and I don't think they are legacy, so renaming them to follow this new pattern is not something I planned for. Which leaves the option for this code to follow the already established pattern (by 3 other examples).

Reply:
Why did we name everything notification? Notifications are very much associated with mobile providers and Stripe is explicit in naming them events. All of the Radom literature refers to them as events and when we discuss them they are referred to as events.

Reply:
> Why did we name everything notification?

Good question. That's historical stuff, and when I was working on the mentioned areas of code, there was not enough resources to also do renaming along with fixes. So we got more examples of notifications. Why is that? Because in those changes I chose consistency, which is not always great, but at least helps make some of it easier to reason about. I agree, those are events really, and should have been called like that in the first place, but they weren't.

And now we're in a situation where we have got 3 examples of using 1 terminology, and this PR is adding a fourth but that uses a new terminology. This is inconsistent. If there's 3 or more of something, the subsequent examples should follow the lead, generally speaking. If a new one is different, it sends a message that this signifies the start of moving to the new terminology. And since there has not been a conversation about doing it, I asked made the comment and suggestion. Which is to say that we should either decide to adopt a new terminology, which means updating old code as well, or the new code should for the time being adhere to the existing one.

We can discuss this offline as well.